OpenRA.Mods.Common/Traits/Buildings/TransformsIntoTransforms.cs

```diff
@@ -44,7 +43,7 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 			if (!order.Queued)
 				currentTransform.NextActivity?.Cancel(self);
 
-			currentTransform.Queue(new IssueOrderAfterTransform("DeployTransform", order.Target, Color.Green));
+			currentTransform.Queue(new IssueOrderAfterTransform("DeployTransform", order.Target));
```

Comment:
Why is this one not needed?

Reply:
Deploying happens at the actor's current location, so doesn't have a target line.